convert/convert.go

```diff
@@ -66,17 +91,62 @@ type Converter interface {
 	writeFile(io.WriteSeeker, llm.KV, []llm.Tensor) error
 }
 
+type AdapterConverter interface {
+	KV(llm.KV) llm.KV
+	Tensors([]Tensor) []llm.Tensor
+
+	tensorName(string) string
```

Comment:
https://github.com/ollama/ollama/pull/5365 changes how tensor names are mapped so be aware